portfolio/src/main/webapp/index.html

```diff
@@ -98,51 +97,45 @@ <h2>Hobbies</h2>
       <div class="container">
         <div class="row">
           <div class="about">
-            <center>
-              <h2>My Favorites</h2>
-              <!-- TODO: Change to unordered list & keep consistent formatting -->
-              <p class="struct">
-                Long Drives<br>
-                Avocados<br>
-                Lavendar<br>
-                Sweater Weather<br>
-              </p>
+            <h2>My Favorites</h2>
+            <p class="struct">
+              Long Drives<br />
+              Avocados<br />
+              Lavendar<br />
+              Sweater Weather<br />
+            </p>
           </div>
-          </center>
           <div class="image">
-            <center><img class="img-responsive" src="images/favorites.png" height="250" width="250"/></center>
+            <img src="images/favorites.png" alt="Favorites" height="250" width="250" />
           </div>
         </div>
       </div>
     </div>
+    <!-- Map -->
+    <div id="map" class="map_format"></div>
     <!-- Generate Random Quote -->
     <div class="info-row info-row1">
-      <center>
-        <p class="struct">
-          <body onload="getRandomQuote()">
+      <body onload="getRandomQuote()">
```

Comment:
You can't have a `<body>` inside a `<div>`, could you remove the enclosing div?

Reply:
I wanted a specific format for the quotes so when I remove the <div> the formatting goes away.

Reply:
Could you set the class on the body?

Reply:
ok sure